### DISCUSSION TITLE
Installation

### DISCUSSION THREAD

**Discussion by thomasbxl** [fw4-bvba/whise-api#13] [Q&A]
Bonjour,
Je découvre votre API, qui pourrait bien m'aider... mais je ne suis pas un habitué des API (oui je sais il serait temps de s'y mettre...).
J'utilisais websiteservices pour Whise, mais la avec leur nouveau système je suis un peu perdu... je ne dois faire que de l'affichage de biens (pas de gestion de contact, calendrier,...).
Il me demande d'utiliser Postman, mais ils sont occupés, car mes accès ne me donnent pas le Token.
En attendant, j'aimerais essayer d'installer votre API, mais j'ai déjà du mal à voir quels fichiers je dois mettre dans l'hébergement et comment faire les appels/connexion de départ. Après vu vos explications cela m'a l'air claire pour avoir les infos... c'est juste l'installation de départ ou je bug....
Désolé de le faire en francais, je vais quand même mettre la traduction Google...

---

Hello,
I discovered your API, which could help me... but I'm not used to APIs (yes I know it's time to get started...).
I used websiteservices for Whise, but with their new system I'm a little lost... I just have to display the properties for sale (no contact management, calendar,...).
He asks me to use Postman, but they are busy, because but access does not give me the Token.
In the meantime, I'd like to try out your API, but I'm already struggling to see what files I need to put in the hosting and how to make the outgoing calls. After seeing your explanations, it seems clear to me to get the info... it's just the initial installation where I'm bugging...

la traduction à l'air très farfelue... :) 

Merci
Thomas.

**Comment by fyrts** [comment]
First step would be to sign up for a Website Designer Marketplace Supplier account at https://web.whise.eu/sign-up-marketplace.html
Once that's done, you can log in and view a list of your clients at https://web.whise.eu/marketplace-admin.html, there can click on "Get token" to get an access token. Initially you'll only have access to a single client for testing purposes. To request access for a specific client, send an email to api@whise.eu (mention the username of your Marketplace Supplier account, and CC the client). To authenticate your requests to the API you need to put the access token in the Authorization header as mentioned on http://api.whise.eu/WebsiteDesigner.html#section/Authentication

If you would like to use the PHP package, I strongly recommend using Composer (https://getcomposer.org/doc/00-intro.md). It may seem like a hassle to set up initially, but it's a genuine help in the long run.

Should you run into any more issues dealing with the API, I'll do my best to help. I don't work for Whise, but I do have quite a bit of experience dealing with them.

**Reply by thomasbxl** [reply]
Thanks !
I will read that.

I have all access and they are working now.
I get the token...

I hope to find the solution now with your information.
Otherwise I'll come back to you... :)
Thanks.

**Reply by thomasbxl** [reply]
Damn... the hosting is shared by ovh... you need a VPS...I think...

**Reply by fyrts** [reply]
If you can't use Composer in your production environment, you could use Composer locally and upload the `vendor` directory.

**Reply by thomasbxl** [reply]
Thanks.
But I will see if we can not use CURL. Just for display.
I would be surprised if all their customers had to change hosting...

**Reply by thomasbxl** [reply]
It's ok, with CURL I was able to get the 2 tokens and display the json with all ads...
Thanks for taking time to respond to me...
and sorry for my English... :o)

**Reply by fyrts** [reply]
It's fine! I'm glad you got it to work.
